api/tests.py

```diff
@@ -50,77 +50,51 @@ def test_get_matches_basic(self):
         self.assertEqual(res.status_code, 200)
-        self.assertCountEqual(
-            res.json(), [{"username": "dig"}, {"username": "fog"}, {"username": "gil"}]
-        )
+        self.assertCountEqual(usernames(res.json()), ("dig", "fog", "gil"))
         self.client.login(username="cad", password="password")
         res = self.client.get("/api/match/", format="json")
         self.assertEqual(res.status_code, 200)
         self.assertCountEqual(
-            res.json(),
-            [
-                {"username": "dig"},
-                {"username": "eel"},
-                {"username": "fog"},
-                {"username": "gil"},
-            ],
+            usernames(res.json()),
+            ("dig", "eel", "fog", "gil"),
         )
         self.client.login(username="dig", password="password")
         res = self.client.get("/api/match/", format="json")
         self.assertEqual(res.status_code, 200)
         self.assertCountEqual(
-            res.json(),
-            [
-                {"username": "ale"},
-                {"username": "cad"},
-                {"username": "eel"},
-                {"username": "fog"},
-                {"username": "gil"},
-            ],
+            usernames(res.json()),
+            ("ale", "cad", "eel", "fog", "gil"),
         )
         self.client.login(username="eel", password="password")
         res = self.client.get("/api/match/", format="json")
         self.assertEqual(res.status_code, 200)
         self.assertCountEqual(
-            res.json(),
-            [
-                {"username": "cad"},
-                {"username": "dig"},
-                {"username": "fog"},
-                {"username": "gil"},
-            ],
+            usernames(res.json()),
+            ("cad", "dig", "fog", "gil"),
         )
         self.client.login(username="fog", password="password")
         res = self.client.get("/api/match/", format="json")
         self.assertEqual(res.status_code, 200)
         self.assertCountEqual(
-            res.json(),
-            [
-                {"username": "boy"},
-                {"username": "cad"},
-                {"username": "dig"},
-                {"username": "eel"},
-                {"username": "gil"},
-            ],
+            usernames(res.json()),
+            ("boy", "cad", "dig", "eel", "gil"),
         )
         self.client.login(username="gil", password="password")
         res = self.client.get("/api/match/", format="json")
         self.assertEqual(res.status_code, 200)
         self.assertCountEqual(
-            res.json(),
-            [
-                {"username": "ale"},
-                {"username": "boy"},
-                {"username": "cad"},
-                {"username": "dig"},
-                {"username": "eel"},
-                {"username": "fog"},
-            ],
+            usernames(res.json()),
+            ("ale", "boy", "cad", "dig", "eel", "fog"),
```

Comment:
Not sure if this is important for this PR but I tried running the test and the below is what I got:
<img width="776" alt="Screen Shot 2021-02-18 at 7 30 11 PM" src="https://user-images.githubusercontent.com/28907798/108439829-03034d00-7220-11eb-89f7-5f73ce4ef094.png">

Reply:
This is a problem with python 3.8. [Python 3.9](https://docs.python.org/3/whatsnew/3.9.html#type-hinting-generics-in-standard-collections) allows type hints to use the respective collection now. Previously, you had to import a special object from the typings module.